### ISSUE TITLE
Getting "error: Unexpected token I in JSON at position 0" when executing azbb from MacOS

### ISSUE THREAD

**Issue by prwitt** [mspnp/template-building-blocks#304]
hey guys,

When trying to deploy resources via azbb from a MacOS (Sierra 10.12.6), I'm getting the following error:

`azbb --location westus --resource-group rg-azbb --parameters-file deploy-vnet.json

"error: Unexpected token I in JSON at position 0"`

It looks like the JSON file is invalid, even though the same JSON file works well on Windows.

This is the JSON file I tried:

```
{
    "$schema": "https://raw.githubusercontent.com/mspnp/template-building-blocks/spikes/spikes/nodejs-spike/schemas/buildingBlocks.json",
    "contentVersion": "1.0.0.0",
    "parameters": {
        "buildingBlocks": {
            "value": [{
                "type": "VirtualNetwork",
                "settings": [{
                    "name": "virtual-network-1",
                    "addressPrefixes": ["10.0.0.0/16"],
                    "subnets": [{
                            "name": "subnet-1",
                            "addressPrefix": "10.0.1.0/24"
                        },
                        {
                            "name": "subnet-2",
                            "addressPrefix": "10.0.2.0/24"
                        }
                    ]
                }]
            }]
        }
    }
}

```

Thanks,
--pr

**Comment by bennage** [comment]
We're looking into it.

**Comment by bennage** [comment]
Can you attach the actual file?

**Comment by prwitt** [comment]
Sure, please, find attached.

[deploy2-vnet.json.zip](https://github.com/mspnp/template-building-blocks/files/1272401/deploy2-vnet.json.zip)

Thanks

**Comment by telmosampaio** [comment]
prwitt, can you check the npm and nodejs versions running on your Mac, please?

**Comment by prwitt** [comment]
hi Telmo,

sure:
npm --version
5.3.0
node --version
v8.1.4

Thanks,
--Paulo Renato

**Comment by bennage** [comment]
I cannot repro using the file on:
- Node v6.11.2
- macOS 10.12.6

**Comment by telmosampaio** [comment]
Interesting... Paulo, any chance you could test it on the latest LTS version for node? That should be 6.11.3.

**Comment by bennage** [comment]
I was not able to repro with Node v8.4.0.

**Comment by prwitt** [comment]
hi Telmo,

I installed node 6.11.3 and I'm still seeing the same issue:

bash-3.2$ npm --version
3.10.10
bash-3.2$ node --version
v6.11.3
bash-3.2$ azbb --subscription-id 30ec953a-f038-4636-b9b6-4ff8ba87b572 --location westus --resource-group rg-azbb --parameters-file deploy2-vnet.json

  error: Unexpected token I in JSON at position 0


thanks,
--Paulo Renato

**Comment by telmosampaio** [comment]
Ok, I'm thinking this might be a line break/carriage return issue. Can we schedule a Skype session to go over it with you, on your machine?

**Comment by prwitt** [comment]
sure, will schedule the session with you.
Thanks Telmo!

**Comment by prwitt** [comment]
Thanks for your time guys, we identified a workaround.
The issue was related to the Azure CLI output that in my case was set to table. 
As a temporary fix, after moving it to JSON as suggested during the troubleshooting call w/ Andrew, it worked well.

Regards,
--pr

**Comment by atoakley** [comment]
Fixed in PR #315
